src/conversion.jl

```diff
@@ -34,13 +33,13 @@ GI.ngeom(::GI.PolygonTrait, p::Polygon) = length(rings(p))
 GI.getgeom(::GI.PolygonTrait, p::Polygon, i) = rings(p)[i]
 
 GI.ncoord(::GI.AbstractGeometryTrait, m::Multi) = embeddim(m)
-GI.ngeom(::GI.AbstractGeometryTrait, m::Multi) = length(collect(m))
-GI.getgeom(::GI.AbstractGeometryTrait, m::Multi, i) = collect(m)[i]
+GI.ngeom(::GI.AbstractGeometryTrait, m::Multi) = length(parent(m))
+GI.getgeom(::GI.AbstractGeometryTrait, m::Multi, i) = parent(m)[i]
 
-GI.isfeaturecollection(::Data) = true
-GI.trait(::Data) = GI.FeatureCollectionTrait()
-GI.nfeature(::Any, d::Data) = nitems(d)
-GI.getfeature(::Any, d::Data, i) = d[i, :]
+GI.isfeaturecollection(::Type{<:AbstractGeoTable}) = true
```

Comment:
So this function expects users of the interface to use the type instead of the object?